soweego/linker/workflow.py

```diff
@@ -302,6 +311,18 @@ def _shared_preprocessing(df, will_handle_dates):
     return df
 
 
+def _normalize_values(values):
+    normalized_values = set()
+    if values is nan or not all(values):
```

Comment:
The `not all(values)` fails when one of the values is `None`, instead of returning `False` when _all_ the values are None, as intended. This check should actually be done with `any()`: 

```suggestion
    if values is nan or not any(values):
```